packages/core/src/sink/SafeSink.ts

```diff
@@ -1,34 +1,38 @@
 /** @license MIT License (c) copyright 2010-2016 original author or authors */
 /** @author Brian Cavalier */
 /** @author John Hann */
+import { Sink, Time } from '@most/types'
 
-export default class SafeSink {
-  constructor (sink) {
+export default class SafeSink<A> implements Sink<A> {
+  private readonly sink: Sink<A>
```

Comment:
Making the `sink` private eliminates subclasses from accessing it. Is this what we want? @briancavalier ?

Reply:
I always add most strict access modifiers by default unless its required to relax constraints for other code